spec/fixtures/storage_root01/moab_storage_trunk/bj/102/hs/9687/bj102hs9687/v0003/manifests/manifestInventory.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- objectId matches druid implied in file path -->
```

Comment:
the comment is in this file (for druid `bj102hs9687`), but it appears that the druid being used in the specs is `bz514sm9647`.  comment here is still totally valid, of course, but i guess i'd have expected the new test and the comment to correspond?